lib/hanami/config/security.rb

```diff
@@ -50,7 +50,7 @@ def content_security_policy(value = nil)
         if value.nil?
           @content_security_policy
         else
-          @content_security_policy = value
+          @content_security_policy = value.split(';').map(&:strip).join('; ')
```

Comment:
Is there any validation on what directives you're going to allow? I'm not 100% of your implementation idea so this may not actually be a concern :stuck_out_tongue: